### DISCUSSION TITLE
How to restart lxqt-panel?

### DISCUSSION THREAD

**Discussion by ailtonbsj** [lxqt/lxqt#2023] [Q&A] [answered]
Hi,

I used LXDE and the command `lxpanelctl restart` with postinst files deb installers.
This command works well. But now I'm in LXQt and can't find some way to do this task.

Using `lxqt-panel &` after kill all instances works on current user, but on post installers not.
Even if I add some $DISPLAY or $XAUTHORITY don't load all parameters of desktop.

Is there any way to restart panel like LXDE ? I can't find this on documentations.

**Comment by tsujan** [comment, marked as answer]
LXQt Session Settings is the tool you can use for starting/stopping LXQt modules but if you want the command-lines, you could find them by using `qdbusviewer`: it contains all the needed info

In the case of `lxqt-panel`, the command-lines are:

```
qdbus org.lxqt.session /LXQtSession startModule lxqt-panel.desktop
qdbus org.lxqt.session /LXQtSession stopModule lxqt-panel.desktop
```

BTW, please don't use the issues section for questions. We have [Discussions](https://github.com/lxqt/lxqt/discussions) for that.
